marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/EpochState.hs

```diff
@@ -163,14 +162,15 @@ test = integration . HE.runFinallies . TN.workspace "chairman" $ \tempAbsPath ->
          in indexerWorker `catch` handleException :: IO ()
 
   found <- liftIO IO.newEmptyMVar
-  liftIO $ (IO.link =<<) $ IO.async $ forever $ do
-    (_, event) <- IO.readChan indexedTxs
-    case Map.lookup poolVKey (EpochState.epochStateEventSDD event) of
-      Just lovelace ->
-        when (lovelace == totalStakedLovelace) $
-          IO.putMVar found (EpochState.epochStateEventEpochNo event) -- Event found!
-      Nothing ->
-        return ()
+  -- TODO Uncomment once we support notifications.
```

Comment:
@berewt Reason I commented this:

In order to support this part, I need the callback to be `(State h, StorableEvent h) -> IO ()`. However, I had to change the callback of `EpochState` to just `State h -> IO ()`, because we need to call that callback when a rewind takes place, thus we have no `StorableEvent` to give.

Alternative, we can change the callback to `(State h, Maybe (StorableEvent h)) -> IO ()`, but feels adhoc. Maybe I can do that temporarily. As we're thinking of supporting notifications, this would be a good scenario here.

What's more, this test is never run in our CI, because of some issue with `cardano-testnet`. So we have no idea if this test actually works.

What do you think?

Reply:
So I'll keep this part commented?